crates/common/types/transaction.rs

```diff
@@ -1352,6 +1451,8 @@ mod mempool {
 #[cfg(test)]
 mod tests {
 
+    use std::str::FromStr;
+
```

Comment:
```suggestion

```

Reply:
yep noticed the CI error. Curious why cargo clippy didn't give me this warning locally.